src/Files.App/Views/Pages/Properties.xaml

```diff
@@ -2,174 +2,182 @@
-						AccessKey="S"
-						Content="{helpers:ResourceString Name=Shortcut}"
-						CornerRadius="0"
-						Tag="Shortcut"
-						Visibility="Collapsed">
-						<NavigationViewItem.Icon>
-							<FontIcon FontFamily="{StaticResource CustomGlyph}" Glyph="&#xF10A;" />
-						</NavigationViewItem.Icon>
-					</NavigationViewItem>
-					<NavigationViewItem
-						x:Name="TabLibrary"
-						AccessKey="L"
-						Content="{helpers:ResourceString Name=Library}"
-						CornerRadius="0"
-						Tag="Library"
-						Visibility="Collapsed">
-						<NavigationViewItem.Icon>
-							<FontIcon Glyph="&#xE1D3;" />
-						</NavigationViewItem.Icon>
-					</NavigationViewItem>
-					<NavigationViewItem
-						x:Name="TabDetails"
-						AccessKey="D"
-						Content="{helpers:ResourceString Name=Details}"
-						CornerRadius="0"
-						Tag="Details"
-						Visibility="Collapsed">
-						<NavigationViewItem.Icon>
-							<FontIcon Glyph="&#xE946;" />
-						</NavigationViewItem.Icon>
-					</NavigationViewItem>
-					<NavigationViewItem
-						x:Name="TabCustomization"
-						Content="{helpers:ResourceString Name=Customization}"
-						CornerRadius="0"
-						Tag="Customization"
-						Visibility="Collapsed">
-						<NavigationViewItem.Icon>
-							<FontIcon Glyph="&#xE771;" />
-						</NavigationViewItem.Icon>
-					</NavigationViewItem>
+			<StackPanel
+				Padding="12,0"
+				Orientation="Horizontal"
+				Spacing="8">
+
+				<Image
+					Width="20"
+					Height="20"
+					Source="ms-appx:///Assets/AppTiles/Dev/Square44x44Logo.png" />
+
+				<TextBlock
+					VerticalAlignment="Center"
+					Style="{StaticResource CaptionTextBlockStyle}"
+					Text="{helpers:ResourceString Name=PropertiesTitle}" />
+
+			</StackPanel>
+		</Border>
+
+		<!--  Main property window's NavigationView  -->
+		<NavigationView
+			x:Name="MainPropertyNavigationView"
+			Grid.Row="1"
+			Width="80"
+			Margin="0,-4,0,2"
+			HorizontalAlignment="Left"
+			IsBackButtonVisible="Collapsed"
+			IsPaneToggleButtonVisible="False"
+			IsSettingsVisible="False"
+			IsTitleBarAutoPaddingEnabled="False"
+			MenuItemsSource="{x:Bind NavViewItems, Mode=TwoWay}"
+			OpenPaneLength="88"
+			PaneDisplayMode="Left"
+			SelectionChanged="NavigationView_SelectionChanged">
+			<!--  TODO: Use ResourceDictionary to customize them and avoid to use negative margins  -->
+
+			<!--  Porperty menu item template  -->
+			<NavigationView.MenuItemTemplate>
+				<DataTemplate x:DataType="local:SquareNavViewItem">
 					<NavigationViewItem
-						x:Name="TabCompatibility"
-						Content="{helpers:ResourceString Name=Compatibility}"
-						CornerRadius="0"
-						Tag="Compatibility"
-						Visibility="Collapsed">
-						<NavigationViewItem.Icon>
-							<FontIcon Glyph="&#xECAA;" />
-						</NavigationViewItem.Icon>
+						Height="62"
+						IsSelected="{x:Bind IsSelected, Mode=TwoWay}"
+						Tag="{x:Bind ItemType}">
+						<NavigationViewItem.Content>
+							<StackPanel Margin="-8,0,-12,0" Spacing="6">
```

Comment:
Spacing should be 4 or 8.

Reply:
If do so, it doesn't looks familiar with the MS Store